api/metrics-api-jaxrs/pom.xml

```diff
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hawkular-metrics-parent</artifactId>
     <groupId>org.hawkular.metrics</groupId>
-    <version>0.3.0-SNAPSHOT</version>
+    <version>0.3.1-SNAPSHOT</version>
```

Comment:
Snapshots are published automatically to Nexus, I do not want to force depending projects to upgrade overnight to the new API. There will be a few differences, so it is better to bump the version and let each project migrate at a later time.